src/test/java/org/mskcc/smile/ValidRequestCheckerTest.java

```diff
@@ -2,7 +2,7 @@
 
 import com.fasterxml.jackson.databind.ObjectMapper;
 import java.util.Map;
-import junit.framework.Assert;
+import org.junit.Assert;
```

Comment:
The previous Assert class has been deprecated and moved from `junit.framework` to `org.junit` in JUnit 4.